pkg/helm/charts/awslbc/awslbc.go

```diff
@@ -0,0 +1,196 @@
+// Package awslbc provides a Helm chart for installing:
+// - https://github.com/kubernetes-sigs/aws-load-balancer-controller
+// - https://github.com/aws/eks-charts/tree/master/stable/aws-load-balancer-controller
+package awslbc
+
+import (
+	"time"
+
+	"github.com/oslokommune/okctl/pkg/helm"
+)
+
+// New returns an initialised Helm chart for installing aws-alb-ingress-controller
+func New(values *Values) *helm.Chart {
+	return &helm.Chart{
+		RepositoryName: "eks",
+		RepositoryURL:  "https://aws.github.io/eks-charts",
+		ReleaseName:    "aws-load-balancer-controller",
+		Version:        "1.1.3",
+		Chart:          "aws-load-balancer-controller",
+		Namespace:      "kube-system",
+		Timeout:        5 * time.Minute, // nolint: gomnd
+		Values:         values,
+	}
+}
+
+// NewDefaultValues returns the mapped values.yml containing the default
+// values
+func NewDefaultValues(clusterName, vpcID, region string) *Values {
+	return &Values{
+		ReplicaCount:                  1, // nolint: gomnd
+		ClusterName:                   clusterName,
+		Region:                        region,
+		VpcID:                         vpcID,
+		TerminationGracePeriodSeconds: 10, // nolint: gomnd
+		EnableCertManager:             false,
+		ServiceAccount: ServiceAccount{
+			Create: false,
+			Name:   "aws-load-balancer-controller", // this is too fragile, refactor
```

Comment:
Refactor before or after merge?

Reply:
We are doing this many places already 😭 we should fix this, but perhaps as its own PR, so we can fix it at all places?